partcad/src/partcad/test/cad.py

```
@@ -0,0 +1,24 @@
+# Licensed under Apache License, Version 2.0.
+#
+
+from .test import Test
+from .. import logging as pc_logging
+
+
+class CadTest(Test):
+    def __init__(self):
+        super().__init__("cad")
+
+    async def test(self, ctx, shape):
```

Ruff (0.8.2)

Missing return type annotation for public function `test`

Add return type annotation: `None`

(ANN201)